packages/frames.js/src/getFrameFlattened.ts

```diff
@@ -42,6 +43,7 @@ export function getFrameFlattened(frame: Frame): FrameFlattened {
       : {};
 
   const metadata: FrameFlattened = {
+    [`og:image`]: frame.ogImage || frame.image,
```

Comment:
this might interfere with the `generateMetadata`'s `openGraph` fields... https://github.com/framesjs/frames.js/pull/302

Reply:
Nope, there is no problem with that.

Reply:
Do you know how it behaves when `other["og:image"]` and `openGraph["images"]` is specified?

https://nextjs.org/docs/app/api-reference/functions/generate-metadata#opengraph

Reply:
I didn't try how it works if you use both, but it works when you supply `og:image` to `other` as well. I believe that when you do that it will overwrite the value in `openGraph` but I haven't tried.

In my opinion `openGraph` is there just to make it more convenient for developers and add some type hints.

Reply:
I think it should be really easy to try it locally with both and see how it works.

Reply:
yeah will do